server/routes/settings/index.ts

```diff
@@ -223,6 +225,80 @@ settingsRoutes.post('/plex/sync', (req, res) => {
   return res.status(200).json(jobPlexFullSync.status());
 });
 
+settingsRoutes.get('/jellyfin', (_req, res) => {
+  const settings = getSettings();
+
+  //DO NOT RETURN ADMIN USER CREDENTIALS!!
+  res.status(200).json(omit(settings.jellyfin, ['adminUser', 'adminPass']));
+});
+
+settingsRoutes.post('/jellyfin', (req, res) => {
+  const settings = getSettings();
+
+  settings.jellyfin = merge(settings.jellyfin, req.body);
+  settings.save();
+
+  return res.status(200).json(settings.jellyfin);
+});
+
+settingsRoutes.get('/jellyfin/library', async (req, res) => {
+  const settings = getSettings();
+
+  if (req.query.sync) {
+    const userRepository = getRepository(User);
+    const admin = await userRepository.findOneOrFail({
+      select: ['id', 'jellyfinAuthToken'],
+      order: { id: 'ASC' },
+    });
+    const jellyfinClient = new JellyfinAPI(
+      settings.jellyfin.hostname ?? '',
+      admin.jellyfinAuthToken ?? ''
+    );
+
+    const libraries = await jellyfinClient.getLibraries();
+
+    const newLibraries: Library[] = libraries
+      // Remove libraries that are not movie or show
+      .filter((library) => library.type === 'movie' || library.type === 'show')
```

Comment:
Don't need this filtering here since we already do it in `server/api/jellyfin`'s `getLibraries` method.